.travis.yml

```diff
@@ -2,18 +2,20 @@ language: python
 
 matrix:
   include:
-    - python: 2.7
-      env: TOX_ENV=py27
-    - python: 2.7
-      env: TOX_ENV=py27-django18
-    - python: 2.7
-      env: TOX_ENV=py27-django111
-    - python: 3.6
-      env: TOX_ENV=py36
-    - python: 3.6
-      env: TOX_ENV=py36-django18
-    - python: 3.6
-      env: TOX_ENV=py36-djangostable
+    - python: 3.7
+      env: TOX_ENV=py37-django22
+    - python: 3.7
+      env: TOX_ENV=py37-django32
+    - python: 3.8
+      env: TOX_ENV=py38-django22
+    - python: 3.8
+      env: TOX_ENV=py38-django32
+    - python: 3.9
+      env: TOX_ENV=py39-django22
+    - python: 3.9
+      env: TOX_ENV=py39-django32
+    - python: 3.9
+      env: TOX_ENV=py39-djangostable
```

Comment:
There are things that work with Django 2.2, and 3.2 but not 3.0 (which is deprecated, I know). I'm not saying you HAVE to support 3.0 and 3.1, but just, be aware that you may run into surprises while upgrading, given that the only supported path for upgrading is to go through all intermediate versions.

Reply:
Yeah, you're right. I've added intermediate versions to the test suite, just to be sure we won't get any issues with them.